Sources/DistributedActors/ClusterSystem.swift

```diff
@@ -930,6 +930,11 @@ public extension ClusterSystem {
 
         _ = try await ask.value // discard the _Done
     }
+
+    enum RemoteCall {
```

Comment:
(move to top-level please), and maybe some minor docs like:

```suggestion
/// Allows for configuring of remote calls by setting task-local values around a remote call being made.
/// 
/// ### Example: Override remote call timeouts
/// ```
/// try await RemoteCall.with(timeout: .seconds(1)) { 
///     try await greeter.greet("Caroline")
/// }
/// ```
public enum RemoteCall { // does swiftformat remove that public? grrr
```

I guess we need that "nicer API" as well rather than just exposing the task local hmmm...